pallets/tendermint-light-client/src/types.rs

```diff
@@ -309,6 +351,18 @@ impl TryFrom<TimestampStorage> for Time {
     }
 }
 
+impl From<Time> for TimestampStorage {
```

Comment:
if it can fail the convention is to impl `TryFrom`

Reply:
Addressed in [`a1d4b39` (#361)](https://github.com/Cardinal-Cryptography/aleph-node/pull/361/commits/a1d4b3912d404224dae6891f633adf10079b03f9)